pandas/tests/generic/test_generic.py

```diff
@@ -202,6 +202,12 @@ def test_downcast(self):
         expected = o.astype(np.int64)
         self._compare(result, expected)
 
+    def test_multiple_astype_casts(self):
+        df = DataFrame({'A': [1, 2, 'z'], 'B': [3, 4, 'z']})
+        df.astype({'A': int,
+                   'B': 'datetime64[ns]'},
+                  errors='ignore')
```

Comment:
So the point of tests is to be explicit about expectations. You should assign the result of this last expression to a local variable called `result` and create a separate DataFrame called `expected` with the anticipated values. Then use `tm.assert_frame_equal` to ensure correctness. You'll see this idiom in quite a few places

Also this test is probably better served in pandas/tests/frame/test_dtypes.py

Reply:
I'm aware of `with pytest.raises` but this test is essentially checking the absence of the default behavior of astype errors which defaults to `raise`.   Since there is no exception thrown we succed and know the errors='ignore' override worked.

Reply:
Added in d7c9d5a350ba9e9fd79e0300a5867911d306c600